packages/earth-map/src/components/header/component.tsx

```diff
@@ -127,6 +127,7 @@ const Header = (props: IProps) => {
           onClick={handleResetLocation}
         />
       </Link>
+      {allGroups.length > 0 && <div style={{ display: 'contents' }}>
```

Comment:
https://caniuse.com/#feat=css-display-contents
This has poor support at the moment. Can you refactor it?